### ISSUE TITLE
Video: Video controls are not visible on a video with white background.

### ISSUE THREAD

**Issue by shanelin1** [eBay/ebayui-core#1437]
# Bug Report

## Description
User interface elements must have a contrast ratio of 3:1 with adjacent background.

Actual Result
Controls are white and video background is also white in this case so contrast ratio is approx. 1:1

## Screenshots
![image](https://user-images.githubusercontent.com/87088707/124802434-d02ace80-df0c-11eb-91ac-74248afa99b5.png)
